internal/cypress/docker/runner.go

```diff
@@ -71,6 +75,52 @@ func (r *Runner) RunProject() (int, error) {
 	return errorCount, nil
 }
 
+// checkCypressVersionAvailability do several checks before running Cypress tests.
+func (r *Runner) checkCypressVersionAvailability() error {
```

Comment:
Just a future note that we'd likely want to move this into a shared docker lib, since the exact same check will need to be performed by other frameworks as well. All good for now!

Reply:
👍